cmd/provider.go

```diff
@@ -21,53 +21,28 @@ var (
 		Example: `
   # Downloads all providers specified in config.hcl:
   cloudquery provider download
-  # Upgrades all providers specified in config.hcl
-  cloudquery provider upgrade 
-  # Upgrade one or more providers
-  cloudquery provider upgrade aws
-  # Downgrades all providers specified in config.hcl
-  cloudquery provider downgrade 
-  # Downgrades one or more providers
-  cloudquery provider downgrade aws, gcp
+  # Sync (Upgrade or Downgrade) all providers specified in config.hcl This will also create the schema.
+  cloudquery provider sync 
+  # Sync one or more providers
+  cloudquery provider sync aws, gcp
   # Drop provider schema, running fetch again will recreate all tables unless --skip-build-tables is specified
   cloudquery provider drop aws
-  # build provider schema
-  cloudquery provider build-schema aws
 `,
 		Version: core.Version,
 	}
 
-	providerUpgradeHelpMsg = "Upgrades one or more providers schema version based on config.hcl"
-	providerUpgradeCmd     = &cobra.Command{
-		Use:   "upgrade [providers,...]",
-		Short: providerUpgradeHelpMsg,
-		Long:  providerUpgradeHelpMsg,
+	providerSyncHelpMsg = "Sync (Upgrade or Downgrade) all providers specified in config.hcl This will also create the schema"
```

Comment:
```suggestion
	providerSyncHelpMsg = "Download the providers specified in config.hcl and re-create their database schema"
```

[non-blocking] Very subjective - `Downgrade` feels like something I would not want to do (i.e. not expected)